### DISCUSSION TITLE
Where are PDBs saved in Molecular nodes?

### DISCUSSION THREAD

**Discussion by MajaDivjak** [BradyAJohnston/MolecularNodes#255] [General]
Hi Brady,

As you know we have ridiculous firewalls in place where I work and I cannot import PDB structures using Molecular Nodes at the moment. I have access to Blender on another platform and will be able to import the structures there, so was wondering what the file path is please so I can copy the PDB files to my work computer?

Thanks so much and best wishes,

Maja

**Comment by BradyAJohnston** [comment]
Currently the PDB files aren't cached anywhere. Once they are downloaded and loaded in to the `.blend` file they are discarded. They can be cached but I haven't implemented that yet. 

You'll likely just have to download the .pdb files from the PDB on your other computer before transferring them, or transferring the .blend file that you have opened the structures in to.

**Reply by MajaDivjak** [reply]
Ah, I see. So, you can save the PDB files to your computer and then use the Local File button to upload?

**Reply by MajaDivjak** [reply]
Yes, that worked.
Can you offer some advice please on the best PDB format to use? If you go to the entry in the PDB for 1bna, for example, and click download files, there are numerous formats to select from. I just went with PDB Format, but are there preferred formats please?

Thanks so much,

Maja

**Reply by BradyAJohnston** [reply]
Best file will be `PDBx/.mmCIF`, but unless you are downloading really large structures `PDB` should work perfectly fine as well :)

**Reply by MajaDivjak** [reply]
Thank you, good to know!
